### PR TITLE
Overriding system ifcfg settings in ovirt

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

**Comment by dankenigsberg** [comment]
RHEV is a downstream trademark, please use oVirt in your title.

"Modifying oVirt-generated ifcfg files" seems like a clearer message in my opinion, as "system" is a very general term.

**Comment by dankenigsberg** [comment]
@garrett @thatdocslady can you assist @mmirecki with his blog entry? Is that where you should post blog candidates?

**Comment by thatdocslady** [comment]
@bproffitt can you please review this blog post? Let me know if you have questions!

**Comment by thatdocslady** [comment]
@garrett there seems to be a build failure here, can you please investigate? thanks!

File: source/blog/2016-05-25-modify-ifcfg-files.html.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+title: Modifying oVirt-generated ifcfg files
```
Comment (garrett) [inline source/blog/2016-05-25-modify-ifcfg-files.html.md]: This is missing the frontmatter block starter...

Solution: add `---` as the first line.

**Pull request merged**